docs/metalk8s-ui.rst

```diff
@@ -0,0 +1,70 @@
+Metalk8s-UI
+===========
+
+
+General Cluster Requirements
+----------------------------
+
+To handle the metalk8s-ui the only requirement is to have access to the
+bootstrap node.
+It's on the bootstrap node that you will get all the required
+information.
+
+
+.. note:: The following test has been realized on vagrant
+
+
+Gather required information
+----------------------------
+
+On the host:
+
+.. code-block:: shell
+
+   $ vagrant ssh bootstrap
+
+Then on the bootstrap node:
+
+.. code-block:: shell
+
+   $ sudo su
+   $ export KUBECONFIG=/etc/kubernetes/admin.conf
+   $ ip a show eth1
+
+Please note the IP address, then let's get the port number:
+
+.. code-block:: shell
+
+   $ kubectl get service -n kube-system
+
+   NAME                  TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+   metalk8s-ui           NodePort    10.104.61.208   <none>        80:30923/TCP     3h
+
+Please note the port used by metalk8s-ui (here ``30923``)
+
+
+Use Metalk8s-ui
+---------------
+
+Once you have gather all the information, open your web browser then lauch the
+developer tools using "Ctrl+Shift+I".
+In the URL, past the IP address and the port used by metalk8s-ui
+(IP address and port separated by ``:``).
+
+The login page is loaded. in The developer tools, click on the file
+:file:`config.json`
+You will obtain some URLs (see below)::
+
+  {url: "https://172.21.254.14:6443", url_salt: "http://172.21.254.11:4507"}
+  url: "https://172.21.254.14:6443"
+  url_salt: "http://172.21.254.11:4507"
+
+take the first URL address and copy-paste it in a new tab.
+An alert appear concerning a non-private connection, click on the button
+``Advanced`` then click on the link ``Proceed to 172.21.254.14 (unsafe)``.
+Go back on the first tab and reload the page with :kbd:`Ctrl+F5` (clear the cache)
+
+Once the page reloaded, log with the default login / password.
```

Comment:
```suggestion
Once the page reloaded, log in with the default login / password.
```

Also, why not specify what they are here (`admin/admin` is not unsafe to put in documentation IMO)?

Reply:
Correction done & default log in / passwd  added.
THX